lib/elixir/pages/library-guidelines.md

```diff
@@ -200,7 +200,31 @@ The code above says we are only bringing in the functions from `MyLib` so we can
 
 If the module you want to invoke a function on has a long name, such as `SomeLibrary.Namespace.MyLib`, and you find it verbose, you can leverage the `alias/2` special form and still refer to the module as `MyLib`.
 
-While there are situations where `use SomeModule` is necessary, `use` should be skipped when all it does is to `import` or `alias` other modules. In a nutshell, `alias` should be preferred, as it is simpler and clearer than `import`, while `import` is simpler and clearer than `use`.
```

Comment:
I think this clear and unequivocal "in a nutshell" part would be good to keep, even if we then explain a way to use `use` properly if really needed.
I find this kind of simple and clear guideline very useful during discussions to explain that "this might be overkill and not the blessed way", maybe even this one in particular I think.